code/_globalvars/lists/flavor_misc.dm

```diff
@@ -173,8 +173,12 @@ GLOBAL_LIST_INIT(jumpsuitlist, list(PREF_SUIT, PREF_SKIRT))
 #define UPLINK_PDA "PDA"
 #define UPLINK_RADIO "Radio"
 #define UPLINK_PEN "Pen" //like a real spy!
-#define UPLINK_IMPLANT  "Implant"
-GLOBAL_LIST_INIT(uplink_spawn_loc_list, list(UPLINK_PDA, UPLINK_RADIO, UPLINK_PEN, UPLINK_IMPLANT))
+#define UPLINK_IMPLANT "Implant"
+#define UPLINK_IMPLANT_WITH_PRICE "[UPLINK_IMPLANT] (-[UPLINK_IMPLANT_TELECRYSTAL_COST] TC)"
+// What we show to the user
+GLOBAL_LIST_INIT(uplink_spawn_loc_list, list(UPLINK_PDA, UPLINK_RADIO, UPLINK_PEN, UPLINK_IMPLANT_WITH_PRICE))
```

Comment:
You're not using this list. Could also make this single associative list with saved_value = description

Reply:
This list is used on line 1508 in preferences.dm: https://github.com/tgstation/tgstation/pull/58872/files#diff-abde2422921045156657700c33fd411dd0029a7800d7892ccff2b5923ebc5132R1508

Should I remove it from being a global list and turn it into a list() on that line?

Reply:
Also tried to consider the associative list idea - wouldn't really work due to line 490 in preferences_savefile.dm https://github.com/tgstation/tgstation/pull/58872/files#diff-5705df368d810d2ef8e755ea140098cb348b8e1119c392b196f1ed997fee2729R490

Reply:
i think the diff confused him into thinking this is a new list